packages/front-end/components/Features/RuleModal/index.tsx

```diff
@@ -795,7 +795,9 @@ export default function RuleModal({
           />
         ) : null}
 
-        {ruleType === "experiment-ref-new" && experimentType === "experiment"
+        {(ruleType === "experiment-ref-new" &&
+          experimentType === "experiment") ||
+        ruleType === "experiment"
```

Comment:
question: what's the different between `experiment`, `experiment-ref` and `experiment-ref-new`? For my learning in understanding how the codebase has evolved